apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -35,26 +37,35 @@ const PublishNotes: React.FunctionComponent = () => {
     <div className={styles.verticalFlexContainer}>
       {isVisible(visibility) && (
         <div className={styles.customizationContainer}>
-          {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label]) => {
+          {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label, _]) => {
             const InputTag = getInputTag(property);
+
             return (
               <div className={styles.inputContainer} key={property}>
                 <label htmlFor={property}>
                   <StrongText>{label}</StrongText>
                 </label>
-                <InputTag
-                  id={property}
-                  disabled={isDisabled(visibility)}
-                  value={modelCardInfo[property]}
-                  onChange={event =>
-                    dispatch(
-                      setModelCardProperty({
-                        property: property,
-                        value: event.target.value,
-                      })
-                    )
-                  }
-                />
+                {property === 'exampleTopics' && (
+                  <ExampleTopicsInputs
+                    topics={modelCardInfo.exampleTopics}
+                    readOnly={isDisabled(visibility)}
+                  />
+                )}
+                {property !== 'exampleTopics' && (
```

Comment:
minor: instead of checking if property is `exampleTopics` twice, you could just return null for exampleTopics in this `map` function, and stick an `ExampleTopicsInput` at the end? That's similar to what we do in `ModelCardFields`

Reply:
hmm that sounds reasonable...I just took a stab at this, problem is that the header text (`label`) comes from the constant we're mapping over. lmk if I'm missing something

Reply:
Oh yeah, my recommendation was to early return in this map function so you don't render the header text for example topics at all, and instead just add it after the map. something like 
```
<div className={styles.customizationContainer}>
  {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label, _]) => {
    if (property === 'exampleTopics') {
      return null;
    }
    return ( 
      <div className={styles.inputContainer} key={property}>
        ...
      </div>);
  }
  <div className={styles.inputContainer} key={'exampleTopics'}>
    <label ... />
    <ExampleTopicsInput ... />
  </div>
</div>
```

Reply:
Oh fair, yep that's fine then!